OpenTabletDriver.Configurations/Configurations/Huion/H951P.json

```
@@ -0,0 +1,39 @@
+{
+  "Name": "Huion H951P",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 221.0,
+      "Height": 141.0,
+      "MaxX": 44197.0,
+      "MaxY": 27599.0
```

Tried this part again and got some nice numbers, 200 LPMM resolution checks out too. Guess this must be the right config:
```suggestion
      "Width": 221.0,
      "Height": 138.0,
      "MaxX": 44200.0,
      "MaxY": 27600.0
```

---

Thanks for providing these numbers. Does the parser change I suggested work for you?

---

> Does the parser change I suggested work for you?

Left a comment on your suggestion